<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QPcap</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="te">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>391</width>
      <height>441</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbStart">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Захватить пакеты</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>491</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Сортировка</string>
    </property>
    <widget class="QComboBox" name="cb">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>271</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>По длине</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>По времени жизни</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>По адресу получателя</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>По адресу отправителя</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>По длине заголовочной части пакета</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>По контрольной сумме (первый байт контрольной суммы)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>По контрольной сумме (последний байт контрольной суммы)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>По типу инкапсулированного пакета(4 - TCP,  17 - UDP)</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pbSort">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Сортировать</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>271</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>IP</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cbtype">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>271</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>sort()</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Внутренняя</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Внешняя</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>20</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Тип сортировки</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>50</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Протокол</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Сортировка</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pbOpen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть файл</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>pcap_sort v.0.3</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
